### ISSUE TITLE
Emlékeztető

### ISSUE THREAD

**Issue by huncut2016** [Legolaszstudio/novy.vip#13]
Sok sikert, mert nagyon jól halads z?

**Comment by Legolaszstudio** [comment]
Egy todo tőled: "FIX TABLE BORDERS on NovyKréta"

**Comment by Zoldszemesostoros** [comment]
A megosztott fájlok részt a nulláról fejlesztetted le?
Mert nem rossz...

**Comment by Legolaszstudio** [comment]
@Zoldszemesostoros Nem, az apaxy nevű csomagot használtam [(https://github.com/oupala/apaxy)](https://github.com/oupala/apaxy) majd írtam hozzá egy saját témát

**Comment by Zoldszemesostoros** [comment]
És min hostolod az oldalt?

**Comment by Legolaszstudio** [comment]
@Zoldszemesostoros Itthon fut egy kis fájlmegosztó szerver ([openmediavault](https://www.openmediavault.org/) alapú), arra telepítettem a legújabb [apache](https://httpd.apache.org/) verziót és [netdata](https://github.com/netdata/netdata) verziót. A [namecheap](https://www.namecheap.com/)-nél vásároltam a domain-t, a DNS-t a [cloudflare](cloudflare.com) szolgáltatja.

**Comment by Legolaszstudio** [comment]
Fizikálisan egy öreg toshiba laptopon fut az egész
